capstone/src/main/java/com/google/sps/AssociationAnalysis.java

```diff
@@ -3,20 +3,24 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
+import opennlp.tools.stemmer.PorterStemmer;
```

Comment:
Have you made sure that you can use this module for commercial use?

Reply:
Yeah I checked its licensed under the Apache License.